### DISCUSSION TITLE
adding background

### DISCUSSION THREAD

**Discussion by georgemilosh** [alshedivat/al-folio#2010] [Q&A] [answered]
Could someone please explain me a quick hack to add background to the website?

**Comment by george-gca** [comment]
Use the css [background-image property](https://www.w3schools.com/cssref/pr_background-image.php) for this.

**Reply by georgemilosh** [reply]
But which file of the repo I need to edit to add this background to? 

Thanks

**Reply by george-gca** [reply, marked as answer]
It depends on where do you want this background to be, since it could be only for the header, for the footer, the main content, or everything. I would recommend you trying it first by editing the html directly in your browser, like [shown in this tutorial](https://www.lucidchart.com/techblog/2018/05/01/live-editing-html-css-chrome-devtools/). Then you will know where exactly you want to put this.
